### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by XhmikosR**
I didn't plan to merge this yet, but we need the extra debug info to investigate the `gzip` issue where sometimes it shows up and another it doesn't.

@jmervine: if you have any better implementation ideas please let me know.

### CODE REVIEW

**Comment by XhmikosR** [comment]
@jmervine: before anything else, we need this so that we can provide enough debug info to the SP guys to solve the gzip failures.

Please review carefully

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ language: node_js
 node_js:
   - "10"
 
+before_script:
+  - curl https://v4.ifconfig.co/
```
Comment (jmervine) [inline .travis.yml]: What’s the purpose of this?

Comment (XhmikosR) [reply .travis.yml]: So that we get the machine's IP for the SP guys.

**Comment by XhmikosR** [comment]
1. provide X-HW when a test fails
2. provide the machine's IP
3. make sure the production headers don't have the debug headers and the opposite

**Comment by jmervine** [comment]
Gotcha. Looks like a good approach.

**Pull request merged**